nixodmlconverter/convert.py

```diff
@@ -466,10 +475,17 @@ def convert(filename, mode='append'):
 def main(args=None):
     parser = docopt(__doc__, argv=args, version=VERSION)
 
-    files = parser['FILE']
-    print(files)
-    for curr_file in files:
-        convert(curr_file)
+    file_or_dir = parser['FILE_OR_DIR']
+    mode = parser['MODE'] if parser['MODE'] else 'append'
```

Comment:
Check if a valid mode has been provided, otherwise switch to default and print a corresponding message.

Reply:
I think it should exit with error if an invalid mode is provided instead of running with a warning.  If we use the stdlib argparse instead of docopt, it supports defining choices for arg values so it does the work for us.  It will also print the valid choices if an invalid one is provided.

Reply:
I opened an issue for further discussion (issue #31).